File: src/com/company/Commercial.java
package com.company;
import java.util.Scanner;

public class Commercial extends Customer {
    public static void getInfo() {
        Scanner scan = new Scanner(System.in);

        System.out.print("Property name: ");
        String propertyName = scan.nextLine();

        System.out.print("Customer name: ");
        customerName = scan.nextLine();

        System.out.print("Customer phone number: ");
        customerPhone = scan.nextLine();

        System.out.print("Customer address: ");
        customerAddress = scan.nextLine();

        System.out.print("Square footage: ");
        squareFootage = scan.nextInt();

        System.out.print("Multi-property discount? (true or false): ");
        boolean multi = scan.nextBoolean();
        double weeklyRate = 5; // per 1000 feet

        double totalWeeklyCharges = (weeklyRate * squareFootage);

        if(multi){
            System.out.println("\n\nYou are receiving a multi-property discount.");
            System.out.println(propertyName);
            System.out.println(customerName);
            System.out.println(customerAddress);
            System.out.println(customerPhone);
            System.out.println(squareFootage);
            System.out.println("Your charges are $" + String.format("%.2f", totalWeeklyCharges * .9));
            System.out.print("\n");

        }
        else {
            System.out.println("You are receiving no discount.");
            System.out.println(propertyName);
            System.out.println(customerName);
            System.out.println(customerAddress);
            System.out.println(customerPhone);
            System.out.println(squareFootage);
            System.out.println("Your charges are $" + String.format("%.2f", totalWeeklyCharges));
            System.out.print("\n");

        }

    }
}

File: src/Commercial.kt
class Commercial(var propertyName: String,
                 customerName: String,
                 customerPhone: String,
                 customerAddress: String,
                 squareFootage: Double) :
    Customer(customerName, customerPhone, customerAddress, squareFootage) {

    init {
        this.propertyName = propertyName
    }

    var multiProperty : Boolean = false
    var rate : Int = 5

    fun billing() {
        val total = if (multiProperty) {
            (rate * squareFootage * .9).toInt()
        } else {
            (rate * squareFootage).toInt()
        }

        println("\n\nProperty name: $propertyName")
        println("Customer name: $customerName")
        println("Customer phone: $customerPhone")
        println("Customer address: $customerAddress")
        println("Square Footage: $squareFootage")
        println("\n\n")
        println("Weekly Total:\t\t$$total")
    }
}